warp/backend.go

```diff
@@ -65,10 +65,12 @@ func (w *warpBackend) AddMessage(unsignedMessage *avalancheWarp.UnsignedMessage)
 
 	copy(signature[:], sig)
 	w.signatureCache.Put(messageID, signature)
+	log.Debug("Added warp message to backend", "messageID", messageID)
```

Comment:
```suggestion
	log.Debug("Adding warp message to backend", "messageID", messageID)
```

this gets committed in the `versiondb` on Accept, so want to make sure this is not past tense and potentially leading to misleading logs if we ever crash on accept.